security/tokens.html.markerb

```diff
@@ -0,0 +1,150 @@
+
+- **token expiry:** When you create a token, include the duration that the token is valid for with the `--expiry` option. In general, you should specify the shortest possible duration for your use case so that you limit access by time as well as scope. Tokens are valid for 175200h0m0s (20 years) by default.
+
+For all options and commands, refer to the [`fly tokens` docs](/docs/flyctl/tokens/).
+
+### Create app-scoped access tokens
+
+App-scoped tokens are limited to managing a single app and its resources.
+
+If you don't specify a name with the `--name` option, then the default name is `flyctl deploy token`.
+
+Create an app-scoped deploy token with a custom name and expiry:
+
+```
+fly tokens create deploy --name <"my token name"> --expiry <duration>
+```
+
+This example creates an app-scoped deploy token called `staging one` that's valid for 48 hours:
+
+```
+fly tokens create deploy --name "staging one" --expiry 48h
+```
+
+#### Create an app-scoped token for SSH
+
+Create a token to SSH into a single app. The SSH token is scoped to only allow SSH access to a specific app and nothing else. To be able to SSH to an app, this token is also allowed to connect to the org’s WireGuard network.
+
+For example:
+
+```
+flyctl tokens create ssh -a my-app > my-app.token.ssh
+```
+
+Use the token to SSH into the app:
+
+```
+FLY_API_TOKEN=$(cat my-app.token.ssh) flyctl ssh console -a my-app
+```
+
+#### Create an app-scoped token to execute commands on Machines
+
+A machine-exec token can execute a restricted set of commands on an app's Machines. You can specify commands on the command line or with the `--command` and `--command-prefix` options. If no command is provided, all commands are allowed.
+
+Create a machine exec token:
+
+```
+fly tokens create machine-exec --command "<exact command and arguments to run>"
+```
+
+### Create org-scoped access tokens
+
+Org-scoped tokens are limited to managing a single org and its resources, including apps.
+
+If you don't specify a name with the `--name` option, then the default name is `Org deploy token`.
+
+Create an org-scoped deploy token with a custom name and duration:
+
+```
+fly tokens create org --name <"my token name"> --expiry <duration>
+```
+
+This example creates an org-scoped deploy token called `prod` that's valid for one week:
+
+```
+fly tokens create org --name "prod" --expiry 168h
+```
+
+You can further limit an org-scoped deploy token by making it read-only, which limits the token access to reading a single org and its resources. 
+
+If you don't specify a name with the --name option, then the default name is Read-only org token.
+
+Create a read-only org-scoped deploy token with a custom name and expiry:
+
+```
+fly tokens create readonly --name <"my token name"> --expiry <duration>
+```
+
+Create a read-only token based on an existing org-scoped token:
+
+```
+fly tokens create readonly --name <"my token name"> --expiry <duration> --from-existing <token ID>
```

Comment:
Does this work? I'd expect you to have to pass the existing token itself rather than it's ID.

Reply:
I could not get this to work at first, since I was misunderstanding how the option worked (spoiler: `--from existing`  doesn't take the token as an argument!) I will update the text